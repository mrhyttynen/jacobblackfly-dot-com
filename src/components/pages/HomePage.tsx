import {
  DivWithWhiteBackgroundAndBlackBordersThin,
  VerticalFlexBox,
} from "../../styles/styles";
import { SearchingEpImage } from "../Images";
import { SearchingEpLinks } from "../MusicLinks";

export const HomePage: React.FC = () => {
  return (
    <VerticalFlexBox>
      <DivWithWhiteBackgroundAndBlackBordersThin>
        Listen to my EP "Searching"
      </DivWithWhiteBackgroundAndBlackBordersThin>
      <SearchingEpImage />
      <SearchingEpLinks />
    </VerticalFlexBox>
  );
};
