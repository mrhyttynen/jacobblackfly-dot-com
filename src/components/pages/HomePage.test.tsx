import React from "react";
import { render, screen } from "@testing-library/react";
import { HomePage } from "./HomePage";
import renderer from "react-test-renderer";

describe("HomePage", () => {
  it("renders page that matches snapshot", () => {
    const tree = renderer.create(<HomePage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
