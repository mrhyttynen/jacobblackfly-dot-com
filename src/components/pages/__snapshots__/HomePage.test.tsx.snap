// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders page that matches snapshot 1`] = `
<div
  className="sc-bcXHqe gkJjOJ"
>
  <div
    className="sc-eDvSVe WyGuv"
  >
    Listen to my EP "Searching"
  </div>
  <div>
    <img
      alt="Man looking into the camera on a pier"
      src="searching-ep-coverphoto.jpeg"
      width="300"
    />
  </div>
  <div
    className="sc-hLBbgP kbgQYO"
  >
    <div
      className="sc-gswNZR cEnbaZ"
    >
      <a
        className="zoom"
        href="https://open.spotify.com/album/54cMaJgpXMyU6zY72I9PYP"
        id="spotify"
      >
        ep_spotify
      </a>
      <a
        className="zoom"
        href="https://youtube.com/playlist?list=OLAK5uy_mM89XmzCk4Wh1qW5YGY4-fC_NoBoq0q3M"
        id="youtube"
      >
        ep_youtube
      </a>
      <a
        className="zoom"
        href="https://music.apple.com/us/album/searching-ep/1822209020"
        id="apple_music"
      >
        ep_apple
      </a>
      <a
        className="zoom"
        href="https://amazon.co.uk/music/player/albums/B0FF56SNZ6"
        id="amazon_music"
      >
        ep_amazon
      </a>
    </div>
  </div>
</div>
`;
